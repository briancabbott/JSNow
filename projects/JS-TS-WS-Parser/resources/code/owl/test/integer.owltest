0
1
.1
10000.0
23e17
18446744073709551615
18446744073709551616
239485790283745098273409587209348759082374590827345293480193249
---
# large numbers become `number`s when they overflow the limits of an int64
x = y*
y =
 number : num
 integer : int
