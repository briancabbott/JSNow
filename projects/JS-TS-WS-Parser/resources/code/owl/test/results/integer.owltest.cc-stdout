x (0 - 126)
  y : INT (0 - 1)
    integer - 0 (0 - 1)
  y : INT (2 - 3)
    integer - 1 (2 - 3)
  y : NUM (4 - 6)
    number - 0.100000 (4 - 6)
  y : NUM (7 - 14)
    number - 10000.000000 (7 - 14)
  y : NUM (15 - 20)
    number - 2300000000000000000.000000 (15 - 20)
  y : INT (21 - 41)
    integer - 18446744073709551615 (21 - 41)
  y : NUM (42 - 62)
    number - 18446744073709551616.000000 (42 - 62)
  y : NUM (63 - 126)
    number - 239485790283745095914255808429689646853532027556105663440486400.000000 (63 - 126)
