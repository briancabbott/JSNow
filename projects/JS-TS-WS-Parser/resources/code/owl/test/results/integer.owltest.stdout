. 0    
  y:int

. 1    
  y:int

. .1   
  y:num

. 10000.0
  y:num--

. 23e17
  y:num

. 18446744073709551615
  y:int---------------

. 18446744073709551616
  y:num---------------

. 239485790283745098273409587209348759082374590827345293480193249
  y:num----------------------------------------------------------
